### ISSUE TITLE
Issue with color badges in collapsible

### ISSUE THREAD

**Issue by Aterbonus** [Dogfalo/materialize#3952]
### Description
When the ```.red``` or ```.blue``` class is used when the badge is in a collapsible, the color background 
exceeds the limits of the badge (And is behind the badge's background).

### Repro Steps
Put the ```.red``` or ```.blue``` class in a badge in a collapsible.

### Screenshots / Codepen
![Issue Badge](https://cloud.githubusercontent.com/assets/15003767/20683349/66b03310-b58a-11e6-95e7-03d021772fb6.png)

**Comment by smileytechguy** [comment]
Can we get your code?

**Comment by Aterbonus** [comment]
The screenshot is from the documentation in [materializecss.com](http://materializecss.com/badges.html). I just added the ```.red``` class with the inspector.

The html in question is:

```html
<ul class="collapsible" data-collapsible="accordion">
  <li>
    <div class="collapsible-header"><span class="new badge red">4</span><i class="material-icons">filter_drama</i>First</div>
     <div class="collapsible-body"><p>Lorem ipsum dolor sit amet.</p></div>
  </li>
  <li>
    <div class="collapsible-header"><span class="badge">1</span><i class="material-icons">place</i>Second</div>
    <div class="collapsible-body"><p>Lorem ipsum dolor sit amet.</p></div>
  </li>
</ul>
```
I have the same issue using materialize  v0.97.8, with this html:

```html
<div id="notificaciones-container">
  <ul data-collapsible="accordion"  id="lista-notificaciones" class="collapsible">
    <li>
       <div class="collapsible-header"><i class="material-icons">new_releases</i>Notificaciones <span data-badge-caption="0" class="cuenta new badge"></span></div>
       <div class="collapsible-body collection">...</div>
    </li>
  </ul>
</div>
```

The class ```.cuenta``` is just used to select the element with jQuery, it doesn't have styles associated.

**Comment by smileytechguy** [comment]
I am not sure why it isn't working, doesn't work for me either.  Works on a normal collection however

**Comment by acburst** [comment]
Fixed in 48f45d6
